### ISSUE TITLE
Error trying to install latest 1.3.1 from the git tag

### ISSUE THREAD

**Issue by n1k0** [mozilla-services/kinto#142]
[Source script](https://github.com/mozilla-services/kinto.js/blob/master/test/scripts/setup-kinto.sh)

```
…
.venv/bin/cliquet --ini config/kinto.ini migrate
Traceback (most recent call last):
  File ".venv/bin/cliquet", line 9, in <module>
    load_entry_point('cliquet==2.4.0.dev0', 'console_scripts', 'cliquet')()
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/cliquet/scripts/cliquet.py", line 48, in main
    env = bootstrap(args.ini_file)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/pyramid/paster.py", line 131, in bootstrap
    app = get_app(config_uri, options=options)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/pyramid/paster.py", line 31, in get_app
    global_conf=options)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 271, in loadobj
    global_conf=global_conf)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 296, in loadcontext
    global_conf=global_conf)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 320, in _loadconfig
    return loader.get_context(object_type, name, global_conf)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 454, in get_context
    section)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 476, in _context_from_use
    object_type, name=use, global_conf=global_conf)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 406, in get_context
    global_conf=global_conf)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 296, in loadcontext
    global_conf=global_conf)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 328, in _loadegg
    return loader.get_context(object_type, name, global_conf)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 620, in get_context
    object_type, name=name)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 640, in find_egg_entry_point
    pkg_resources.require(self.spec)
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 952, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/var/tmp/kinto/.venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 844, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
    pkg_resources.ContextualVersionConflict: (cliquet 2.4.0.dev0 (/var/tmp/kinto/.venv/lib/python2.7/site-packages), Requirement.parse('cliquet==2.3.1'), set(['kinto']))
```

**Comment by n1k0** [comment]
Note: this one was hard to spot as the error is well hidden deep within collapsed sections in travis build reports, eg. [this one](https://travis-ci.org/mozilla-services/kinto.js/builds/71118771).

**Comment by Natim** [comment]
This is because you are installing cliquet master with kinto 1.3.1 that should be using  https://github.com/mozilla-services/kinto/blob/1.3.1/setup.py#L24

**Comment by Natim** [comment]
If you want to use cliquet master, you should use kinto master as well.

**Comment by Natim** [comment]
> from the git tag
I think our ``dev-requirements.txt`` is installing cliquet master.

We should not use ``install-dev`` for this tests.

**Comment by n1k0** [comment]
Look at the script; I'm just checking out the 1.3.1 tag and `make serve`. Shouldn't installing the specific cliquet version a given Kinto version requires be handled automatically?

**Comment by n1k0** [comment]
> We should not use install-dev for this tests.

So could you come with a dedicated Make command I could simply use for tests?

**Comment by Natim** [comment]
I have a fix here: https://github.com/mozilla-services/kinto/pull/143

**Comment by Natim** [comment]
It wont fix the 1.3.1 tag but it will be fixed for next release. We'll also remove the github link in dev-requirements during releases.
